README.rst

```diff
@@ -176,46 +196,169 @@ For flashing the board OpenOCD must be used. You can optionally use gdb
 as a frontend for OpenOCD as described below.
 
 You must first flash a bootstrap rom before flashing an application.
-Assuming the toolchain was unpacked into *$HOME/issm_2016.0.019/*, this can be
+Assuming the toolchain was unpacked into *$HOME/issm_2016/*, this can be
 done with:
 
-``$ cd $HOME/issm_2016.0.019/tools/debugger/openocd``
+``$ cd $HOME/issm_2016/tools/debugger/openocd``
 
 ``$ ./bin/openocd -f scripts/board/quark_d2000_onboard.cfg``
 
+Create a new terminal session at this point
+
 ``$ gdb``
 
 ``(gdb) target remote :3333``
 
 ``(gdb) monitor clk32M 5000``
 
-``(gdb) monitor load_image $PATH_TO_QMSI/build/debug/quark_d2000/rom/quark_d2000_rom.bin 0x0``
+For Quark™ D2000, the following commands are used to flash a ROM and application to the device:
+
+``(gdb) monitor load_image $PATH_TO_QMSI/build/release/quark_d2000/rom/quark_d2000_rom.bin 0x0``
 
-``(gdb) monitor load_image $PATH_TO_QMSI/examples/hello_world/debug/quark_d2000/bin/hello_world.bin 0x00180000``
+``(gdb) monitor load_image $PATH_TO_QMSI/examples/hello_world/release/quark_d2000/x86/bin/hello_world.bin 0x00180000``
+
+For Quark™ C1000, the following commands are used to flash a ROM and application to the device:
```

Comment:
We refer to this as 'Quark SE' in the 'SoC  Targets / SoC Core'  sections. 

There seems to be some differences throughout the doc with reference to  'D2000' 'Quark D2000' 'Quark SE' etc...  If we use 'Intel' or 'Quark' in the name then we have to use the FULL branded name ... but 'D2000' 'SE' or 'SE C1000' should be ok.